src/pages/Facility/billing/invoice/InvoiceShow.tsx

```
@@ -289,22 +298,70 @@ export function InvoiceShow({
+          <BackButton variant="link" className="px-0 justify-start">
+            <ChevronLeft />
             <span>{t("back")}</span>
           </BackButton>
-        </div>
-        <div className="flex gap-2 flex-col sm:flex-row w-full sm:w-auto">
-          <Button variant="outline" asChild>
+          <div className="h-auto w-px bg-gray-300" />
+          <div>
+            <label className="text-gray-700 text-sm font-medium">
+              {t("patient_name")}
+            </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)